### DISCUSSION TITLE
ord server command error 13

### DISCUSSION THREAD

**Discussion by LNOrville** [ordinals/ord#1919] [Q&A]
I was following instructions in the Hell Money Podcast part one. Installing on Ubuntu. Bitcoin started syncing and it was all going great until:

Installed package `ord v0.5.1` (executable `ord`)
orville@orville:~$ which ord
/home/orville/.cargo/bin/ord
orville@orville:~$ ord server
Listening on [http://0.0.0.0:80]
error: Permission denied (os error 13)

What to do?

**Comment by jokie88** [comment]
`sudo -E ./ord server`

**Reply by Scottlexium** [reply]
this worked for me, however to make sure you can run order directly without using path of ./ord , you will need to add it to exports like this `export PATH="$PATH:/root/bin"`
now you can run `ord --version` and `ord server` directly.

**Comment by Scottlexium** [comment]
currently I need a way to run ords with bitcoin-cli inside docker. anyone has a way to help ?
